src/pages/admin/position.jsx

```diff
@@ -86,18 +87,29 @@ export const PositionAdmin = () => {
                                         name = `${role.name}${roleCrew ? " (" + name + ")":""}`
                                     }
     
+                                    
                                     return (
-                                        <SelectableRow title="Trykk for å åpne" onClick={e => {history.push(`/positions/${role.uuid}`)}}>
-                                            <Column consolas flex="9" visible={!visibleUUID}>{role.uuid}</Column>
-                                            <Column flex="4">{(roleCrew?.name ?? "-")}</Column>
-                                            <Column flex="9">{name}</Column>
-                                            <Column flex="2">{role.name ? "Custom" : "System"}</Column>
-                                            <Column flex="2">{role.position_mappings.filter(mapping => !mapping.event_uuid || mapping.event_uuid === currentEvent.uuid).length}</Column>
-                                            <Column flex="2">{role.permissions.length}</Column>
-                                            <Column flex="0 24px"><IconContainer><FontAwesomeIcon icon={faArrowRight}/></IconContainer></Column>
+                                        <SelectableRow title="Trykk for å åpne" onClick={e => {history.push(`/positions/${role.uuid}`)}} key={role.uuid}>
+                                            <Column mqhide consolas flex="9" visible={!visibleUUID}>{role.uuid}</Column>
+                                            <Column flex="4" mqhide>{(roleCrew?.name ?? "-")}</Column>
+                                            <Column flex="9" mqflex="3">{name}</Column>
+                                            <Column flex="2" mqhide>{role.name ? "Custom" : "System"}</Column>
+                                            <Column flex="2" mqflex="1">{role.position_mappings.filter(mapping => !mapping.event_uuid || mapping.event_uuid === currentEvent.uuid).length}</Column>
+                                            <Column flex="2" mqhide>{role.permissions.length}</Column>
+                                            <Column flex="0 24px" mqhide><IconContainer><FontAwesomeIcon icon={faArrowRight}/></IconContainer></Column>
                                         </SelectableRow>
                                     )
                                 })
+                                .sort((a, b) => {
```

Comment:
Sort before you convert to react objects. You can use one map for line 81 and 82, then sort, then map a second time to convert to react.